### PR TITLE
[Test] Add test to validate rootsquash scenario

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by pranavprakash20** [comment]
run log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-URZE3O/

**Review by MohitBis — APPROVED** [review]
LGTM

File: suites/quincy/nfs/tier-0_nfs_ganesha.yaml [hunk 1]
```diff
@@ -258,3 +258,13 @@ tests:
...
         clients: 1
+
+  - test:
+      name: Nfs access file from  with root-squash enabled
+      module: nfs_verify_file_access_with_rootsquash.py
+      desc: Test NFS file access with root-squash enabled
+      polarion-id:
+      abort-on-fail: false
+      config:
+        nfs_version: 4.2
```
Comment (msainiRedhat) [inline suites/quincy/nfs/tier-0_nfs_ganesha.yaml:269]: Default should be 4.1 protocol

**Review by msainiRedhat — CHANGES_REQUESTED** [review]

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by vamahaja — APPROVED** [review]

**Pull request merged**